packages/thirdweb/src/storage/download.ts

```diff
@@ -8,9 +8,9 @@ export type DownloadOptions = Prettify<
   }
 >;
 
+/** @import { createThirdwebClient } from "../client/client.js" */
```

Comment:
what does this do?

Reply:
Let's us link the type without needing to import it in the actual code

https://devblogs.microsoft.com/typescript/announcing-typescript-5-5-beta/

Reply:
Isn't this just for type imports though?

Reply:
oh interesting - wonder how its gonna render on docs.

one simple thing we can do is just link to `https://portal.thirdwerb.com/references/typescript/v5/<theFunction>` with markdown, that i know renders well on portal

Reply:
Yeah let's do that for now. I was afraid of broken links that way but should be fine